feature/gribi/otg_tests/gribi_static_route_on_lemmings_test/BUILD

```diff
@@ -0,0 +1,36 @@
+load("//ops/netops/lab/wbb/testbeds:testbeds.bzl", "featureprofiles_test_suite")
```

Comment:
@gopalakrishv We should not push Build file in Git. It will be created internally once the PR is merged

Reply:
I have removed the BUILD file